stream_alert/rule_processor/handler.py

```diff
@@ -92,7 +92,9 @@ def __init__(self, context, enable_alert_processor=True):
 
         # Firehose client initialization
         self.firehose_client = None
-        StreamThreatIntel.load_intelligence(self.config)
+
+        # Load Threat Intel config
+        StreamThreatIntel.load_config(self.config)
```

Comment:
Hey @chunyong-lin let's create an instance of the `StreamRules` class that gets cached on the `StreamAlert` class as an instance property. And, within the `StreamRules` initializer, you load an instance of this `StreamThreatIntel` class from the config.